cluster-sync/imageio/imageio.yaml

```diff
@@ -22,7 +22,8 @@ spec:
         ports:
         - containerPort: 12345
       - name: fakeovirt
-        image: machacekondra/fakeovirt:test
+        # update back to original image when changes are merged
```

Comment:
How do we track this change?

Reply:
Ondra has pushed the new image - I will update it in a moment.

Reply:
Done.